android/lib/src/androidTest/java/com/ern/api/impl/BasicNavigationApiTest.java

```diff
@@ -332,7 +332,75 @@ public void perform(SampleActivity activity) {
             }
         });
     }
-    
+    @Test
+    public void testBackgroundBackToApi() {
+        final CountDownLatch latch = new CountDownLatch(1);
+        assertNotNull(rule);
```

Comment:
I know this "`assertNotNull(rule);"` is copied from other test cases. So not as part of this PR, but maybe separately we can clean up and remove these type of assertions that do not test the our code. Same applies to `assertNotNull(scenario);` below. It would reduce the lines in the test class without affecting coverage.